source/client/style/component/_tag.scss

```diff
@@ -5,13 +5,26 @@
   margin-bottom: 10px;
   margin-right: 10px;
   border-radius: 4px;
+  border-style: solid;
+  border-color: #DDDDDD;
+  border-width: 2px;
 }
 
 .tag__text {
   height: 100%;
-  line-height: 80px - 5px * 2;
+  line-height: 76px - 5px * 2;
```

Comment:
I think this must have something to do with the border, so if the border is removed, this can be updated as well.